x-pack/filebeat/input/awss3/metrics.go

```diff
@@ -62,6 +64,8 @@ func newInputMetrics(id string, optionalParent *monitoring.Registry) *inputMetri
 	}
 	adapter.NewGoMetrics(reg, "sqs_message_processing_time", adapter.Accept).
 		Register("histogram", metrics.NewHistogram(out.sqsMessageProcessingTime)) //nolint:errcheck // A unique namespace is used so name collisions are impossible.
+	adapter.NewGoMetrics(reg, "sqs_message_delayed_time", adapter.Accept).
```

Comment:
I was looking over naming in other inputs and I think "lag time" would be a consistent name so `sqs_lag_time`.